### PR TITLE
HPCC-20826 Report exception info in dedicated txsummary entries

### PR DESCRIPTION
**Pull request by rpastrana**
- Splits out txsummary exception info

Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:

Forced exception in ESP request handling.
confirmed expected txsummary output

### CODE REVIEW

**Comment by rpastrana** [comment]
@wangkx please review

**Comment by rpastrana** [comment]
Observed TxSummary:
00000124 2018-10-23 15:34:37.283 28297  1235 "TxSummary[activeReqs=2;auth=NA;contLen=71;rcv=1ms;handleHttp=19ms;user=@127.0.0.1;req=POST wsesdlconfig.PUBLISHESDLDEFINITION v1.4;excepttime=3;exceptcode=-1;total=20ms;]"

**Comment by wangkx** [comment]
@rpastrana looks fine.

**Comment by timothyklemm** [comment]
@rpastrana I am not aware of any dependencies on the original format. However, the original format was introduced in 2011 (https://github.com/hpcc-systems/HPCC-Platform/commit/2101246a2776ba997d96bd47bdabb1264b010181#diff-b598213f08547b1e561ead25ac677b96). I would not consider my answer to be authoritative.

**Comment by timothyklemm** [comment]
@rpastrana Just nitpicking, but should "dedicate" be "dedicated" in the commit message?

**Comment by rpastrana** [comment]
@richardkchapman please merge

**Comment by richardkchapman** [comment]
Should this target 7.0.2?

**Comment by rpastrana** [comment]
@richardkchapman re-targeted

File: esp/files/Login.html [hunk 1]
```diff
@@ -96,8 +96,8 @@
                 <p id="loginStr" class="loginStr"></p>
                 <div class="login">
                     <form method="POST" action="/esp/login">
-                        <input id="username" type="text" name="username" placeholder="username" class="loginStr"></input>
```
Comment (richardkchapman) [inline esp/files/Login.html]: This feels like an unrelated change?

File: esp/src/eclwatch/LZBrowseWidget.js [hunk 2]
```diff
@@ -381,7 +381,7 @@ define([
                         var request = domForm.toObject(this.id + formID);
                         var item = selections[0];
                         lang.mixin(request, {
-                            sourceIP: selections[0].DropZone.NetAddress,
```
Comment (richardkchapman) [inline esp/src/eclwatch/LZBrowseWidget.js]: This feels unrelated

**Comment by richardkchapman** [comment]
I think you need to rebase - this has some unrelated commits included

**Comment by rpastrana** [comment]
@richardkchapman yes, the commit history seems off after the re-target, I will rebase my branch to 7.0.2

**Comment by rpastrana** [comment]
@richardkchapman properly rebased and retargeted 7.0.2

**Comment by richardkchapman** [comment]
Not really suitable for a point release. If no-one is parsing this information, then the change is of very minor importance. If anyone is, there are compatibility concerns that suggest 7.2.0 would be a more suitable target.
